app/ts/components/pages/ConfirmTransaction.tsx

```diff
@@ -1,24 +1,49 @@
 import { useState, useEffect } from 'preact/hooks'
-import { ConfirmTransactionDialogState, ConfirmTransactionSimulationBaseData, ExternalPopupMessage, IsConnected } from '../../utils/interceptor-messages.js'
+import { ConfirmTransactionDialogState, ConfirmTransactionSimulationBaseData, ConfirmTransactionTransactionSingleVisualizationArray, ExternalPopupMessage, IsConnected } from '../../utils/interceptor-messages.js'
 import { SimulatedAndVisualizedTransaction, SimulationAndVisualisationResults } from '../../utils/visualizer-types.js'
 import Hint from '../subcomponents/Hint.js'
-import { ExtraDetailsTransactionCard, GasFee, LogAnalysisCard, SimulatedInBlockNumber, TransactionHeader, TransactionsAccountChangesCard } from '../simulationExplaining/SimulationSummary.js'
+import { ExtraDetailsTransactionCard, GasFee, LogAnalysisCard, SimulatedInBlockNumber, TransactionHeader, TransactionHeaderForFailedToSimulate, TransactionsAccountChangesCard } from '../simulationExplaining/SimulationSummary.js'
 import { CenterToPageTextSpinner } from '../subcomponents/Spinner.js'
 import { AddNewAddress } from './AddNewAddress.js'
 import { AddingNewAddressType, AddressBookEntry } from '../../utils/user-interface-types.js'
 import { sendPopupMessageToBackgroundPage } from '../../background/backgroundUtils.js'
-import { formSimulatedAndVisualizedTransaction } from '../formVisualizerResults.js'
-import { addressString } from '../../utils/bigint.js'
 import { SignerLogoText } from '../subcomponents/signers.js'
-import { SmallAddress, WebsiteOriginText } from '../subcomponents/address.js'
 import { Error as ErrorComponent, ErrorCheckBox } from '../subcomponents/Error.js'
-import { QuarantineCodes, TransactionImportanceBlock } from '../simulationExplaining/Transactions.js'
+import { QuarantineCodes, SenderReceiver, TransactionImportanceBlock } from '../simulationExplaining/Transactions.js'
 import { identifyTransaction } from '../simulationExplaining/identifyTransaction.js'
 import { SomeTimeAgo } from '../subcomponents/SomeTimeAgo.js'
 import { TIME_BETWEEN_BLOCKS } from '../../utils/constants.js'
 
+type UnderTransactionsParams = {
+	pendingTransactions: ConfirmTransactionTransactionSingleVisualizationArray
+}
+
+const HALF_HEADER_HEIGHT = 48 / 2
```

Comment:
It feels like this should use a variable defined elsewhere rather than `48`.  That way if we change the header, this adjusts correctly.